build_js.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+git submodule update --init --recursive
+
+DATA_DIR="$(pwd)/data/"
+
+mkdir -p build_js
+cd build_js
+
+cmake ../src \
+    -DBUILD_GUI_VIEWERS=ON \
+    -DBUILD_PYTHON_BINDINGS=OFF \
+    -DBUILD_DATATOOL=OFF \
+    -DBUILD_PTEX_SUPPORT=OFF \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_PREFIX_PATH="$EMSCRIPTEN" \
+    -DTARGET_GLES2=OFF \
```

Comment:
I think `TARGET_GLES2=OFF` would be better set inside the `if(NOT USE_SYSTEM_MAGNUM)` of `dependencies.cmake`.